Meetings/2023-02-28.md

```diff
@@ -5,24 +5,133 @@
 ## Participants
 
 * Michael Aye (MA)
-
+* Ross Beyer (RB)
+* Andrew Annex (AA)
+* Benoit Seignovert (BS)
+* Trent Hare (TH)
+* Chase Million (CM)
+* Christian Tai Udovicic (CTU)
+  
 ## pending AIs
 
 * kick-onwards the procedures for craterstats affiliation
 * MA to ask Benoit and Chase to fill in the poll for earlier meeting time.
-  
+  * Add-on: Benoit wants it as well.
 ## Agenda
 
+### Trent asks: Lessons learned from taking craterstats on as affiliated?
+
+* Is packaging a requirement?
+* MA: PPTC certainly supports folks that don't know packaging
+* AA: We have project template that would be a good start for starting a new package
+* RB: Take away: State somewhere that we highly encourage to contact us early to be helpful before investing time into wrong direction
+
 ### PDART to extend planetarypy to support MSL/M2020 PDS indexes (AA)
 
+AA reports:
+* During Planetary terrain workshop a Brainstorming happened on things that community needs
+* Lots of little tasks required for a full image processing pipeline
+* CM points out that PDART is now PDAR and doesn't support software anymore, there are new funding lines though
+  * RB points out that was just temporarily, the T is back in PDART for 2023
+* Index files still useful, but missing in PDS4 definitions, at least nominally
+  * MA has repeatedly pointed this out to PDS folks
+  * At last discussion, Anne Rough agreed that index files would be useful, at least via a dynamic creation of an index file for a given set of columns.
+  * MA asked Anne Rough and Jordan Padams in OP slack if there was any progress made for index files for PDS4.
+
 ### plio: Affiliate, absorb, or both (cherry-pick)?
 
+* Controlnets are more complicated "next level" to do mosaics then just plotting it in a QGIS
+* GXP control networks to ASP ??
+* RB: Who is plio lead?
+  * Jay Laura with contributions from others
+* (Edit: I have unfortunately no conclusion noted down from this discussion.)
+
 ### pooch
 
 * pooch is nice but absolutely requires file hashes. We don't have them for PDS data?
+  * AA says it does support ops without file hashes
+  * they are used for file integrity
+  * timestamps aren't the safest
+  * MA feels that engineering for data file safety is violating the [80-20 rule](https://dzone.com/articles/applying-8020-rule-software), in that there are the obvious much lower hanging fruits of making folks happy first about getting data easily in the first place, before worrying about the rare downloading error case where a download is corrupted without the downloading processor realizing it.
+  * Additionally, MA needs to check if `pooch` is making it easy to get to the downloaded datafiles, as this is a wish for many if not most datascientists, to be able to deal with the files directly, when necessary.
 
 ### AOB
 
-* Anything to say about the `pvl` issue about line ending? (RB)
-  
+* RB on `pvl` issue about line ending:
+  * (Edit: Ross, could you help me here, I skipped on taking notes) 
```

Comment:
I'm not sure that you need to do anything other than reference the closed Issue, https://github.com/planetarypy/pvl/issues/106